// Copyright (c) 2020 The Orbit Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";

package orbit_client_protos;

message PresetModule {
  repeated uint64 function_hashes = 1;
  repeated uint64 frame_track_function_hashes = 2;
}

message PresetInfo {
  reserved 1;
  map<string, PresetModule> path_to_module = 2;
}
